src/task7/enums/Meal.java

```diff
@@ -0,0 +1,8 @@
+package task7.enums;
+
+public enum Meal {
+    BONE,
+    FISH,
+    RABBIT,
+    MOUSE;
+}
```

Comment:
You can create one class "enums" and put all enums there

Reply:
if I create one class for enums, then my logic for naming animals not to be work, because I use method getQuantityNames